services/tenant-ui/config/default.json

```diff
@@ -28,7 +28,7 @@
       }
     },
     "ariesDetails": {
-      "acapyVersion": "0.8.1",
+      "acapyVersion": "0.8.2",
```

Comment:
Can update this to `0.8.3pre` at this point (not too important, just for display from this config)

Reply:
We could also use (here and everywhere we use an aca-py image) the 0.9.0-rc0 which includes some extra fixes